src/h5web/providers/mock/MockProvider.tsx

```diff
@@ -1,76 +1,16 @@
-import type { ReactNode } from 'react';
+import { ReactNode, useMemo } from 'react';
 import Provider from '../Provider';
-import { mockFilepath } from './metadata';
-import { assertMockDataset, findMockEntity } from './utils';
-import type { GetValueParams } from '../context';
-import ndarray from 'ndarray';
-import { assertPrintableType, assertArrayShape } from '../../guards';
-import unpack from 'ndarray-unpack';
-import type { Primitive } from '../models';
-import type { PrintableType } from '../../vis-packs/core/models';
+import { MockApi } from './api';
 
 interface Props {
-  filepath?: string;
-  slowOnPath?: string;
-  errorOnPath?: string;
```

Comment:
I decided to remove those props and just hard-code the paths in the API. We used to make use of `errorOnPath` in the tests, but now that I've added a dedicated mock entity to test fetching errors, it's no longer needed.